.github/workflows/build-py-dists.yml

```diff
@@ -8,8 +8,7 @@ jobs:
             fail-fast: false
             matrix:
                 os: [windows-latest, macos-latest]
-                python-version: [3.7, 3.8, 3.9, "3.10", "3.11", "3.12"]

```

Comment:
python 3.7 is end-of-life, we already removed it from the test workflow. And `setup-python` couldn't find an ARM64 python 3.7 distribution.